app/assets/javascripts/dap.min.js

```diff
@@ -0,0 +1,2 @@
;adFlag=true}if(adFlag==true){return url+stAdd}else{return url}}function _initYouTubeTracker(){var _iframes=document.getElementsByTagName("iframe");var vArray=0;for(var ytifrm=0;ytifrm<_iframes.length;ytifrm++){_thisVideoObj=_iframes[ytifrm];var _thisSrc=_thisVideoObj.src;if(IsYouTube_fed(_thisSrc)){_thisVideoObj.src=YTUrlHandler_fed(_thisSrc);var youtubeid=youtube_parser_fed(_thisSrc);_thisVideoObj.setAttribute("id",youtubeid);videoArray_fed[vArray]=youtubeid;vArray++}}}function onYouTubePlayerAPIReady(){for(var i=0;i<videoArray_fed.length;i++){playerArray_fed[i]=new YT.Player(videoArray_fed[i],{events:{onReady:onFedPlayerReady,onStateChange:onFedPlayerStateChange}})}}function onFedPlayerReady(event){}function onFedPlayerStateChange(event){var videoURL=event.target.getIframe().getAttribute("src");var videoId=youtube_parser_fed(videoURL);_thisDuration=(parseInt(event.target.getCurrentTime())/parseInt(event.target.getDuration())*100).toFixed();if(typeof onPlayerStateChange!="undefined"){onPlayerStateChange(event)}if(parseInt(event.data)==parseInt(YT.PlayerState.PLAYING)){if(_thisDuration==0){_f33=false;_f66=false;_f90=false}_sendEvent("YouTube Video","play",videoURL,0)}else if(event.data==YT.PlayerState.ENDED){_sendEvent("YouTube Video","finish",videoURL,0)}else if(event.data==YT.PlayerState.PAUSED){_sendEvent("YouTube Video","pause",videoURL,0);var duration=_thisDuration;if(duration<100){var precentage=_thisDuration;if(precentage>0&&precentage<=33&&_f33==false){_sendEvent("YouTube Video","33%",videoURL,0)}else if(precentage>0&&precentage<=66&&_f66==false){_sendEvent("YouTube Video","66%",videoURL,0)}else if(precentage>0&&precentage<=90&&_f90==false){_sendEvent("YouTube Video","90%",videoURL,0)}}}}}function _initIdAssigner(){var _allDocLinks=document.getElementsByTagName("a");for(var sid=0;sid<_allDocLinks.length;sid++){var currentId=_allDocLinks[sid].getAttribute("id");if(currentId==null||currentId==""||currentId==undefined){_allDocLinks[sid].setAttribute("id","anch_"+sid)}}}function _tagClicks(evObj,evCat,evAct,evLbl,evVal){if(evObj.addEventListener){evObj.addEventListener("mousedown",function(){_sendEvent(evCat,evAct,evLbl,evVal)})}else if(evObj.attachEvent){evObj.attachEvent("onmousedown",function(){_sendEvent(evCat,evAct,evLbl,evVal)})}}if(document.addEventListener){document.addEventListener("DOMContentLoaded",function(){if(tObjectCheck!=window["GoogleAnalyticsObject"]){createTracker(false)}oCONFIG.ENHANCED_LINK.toString()=="true"?_initIdAssigner():"";oCONFIG.AUTOTRACKER.toString()=="true"?_initAutoTracker():"";oCONFIG.YOUTUBE.toString()=="true"?_initYouTubeTracker():""})}else if(document.attachEvent){document.attachEvent("onreadystatechange",function(){if(document.readyState==="complete"){if(tObjectCheck!=window["GoogleAnalyticsObject"]){createTracker(false)}oCONFIG.ENHANCED_LINK.toString()=="true"?_initIdAssigner():"";oCONFIG.AUTOTRACKER.toString()=="true"?_initAutoTracker():"";oCONFIG.YOUTUBE.toString()=="true"?_initYouTubeTracker():""}})}
```

Comment:
Line is too long.<br>Missing semicolon.<br>Use '!==' to compare with 'null'.<br>Expected '{' and instead saw 'return'.<br>Expected '{' and instead saw 'if'.<br>Expected '{' and instead saw 'oCONFIG'.<br>Use '!==' to compare with ''.<br>Use '!==' to compare with 'undefined'.<br>['GoogleAnalyticsObject'] is better written in dot notation.<br>Too many errors. (50% scanned).